src/components/BookingForm.vue

```diff
@@ -2,7 +2,26 @@
   <v-container>
     <v-row class="text-center">
       <v-col>
-        <bad-contained-button id="btnBook" :click="sayHello">
+        <v-text-field
```

Comment:
Instead of using v-text-field, we should create separate component out of it, just like we did for the button.
We can name it "bad-text-field" and then use it whenever we need a text field.

Reply:
I would say we should consider it if we gonna be configuring/styling text field per mockups otherwise why would we need a wrapper ?

Can we merge it as a first iteration and then create a wrapper if we need to?